torch/csrc/utils.h

```diff
@@ -41,6 +41,14 @@
     (PyLong_Check(object) ? PyLong_AsLongLong(object) :                        \
     PyInt_Check(object) ? PyInt_AsLong(object) :                               \
     (throw std::runtime_error("Could not parse real"), 0))
+
+#define THPUtils_unpackReal_BOOL(object)                                       \
```

Comment:
you don't need to define a version for py2 vs py3, bools are the same across versions.  Just put this below, outside of the if/endif.